src/ast/types.ts

```diff
@@ -271,6 +271,7 @@ export interface FunctionDeclaration extends BaseDeclaration {
   type: 'FunctionDeclaration';
   // TODO: Add declaration specifiers
   id: Identifier;
+  params: Identifier[];
```

Comment:
I don't think it affects anything now, but will params be different from `Identifiers` in future because `Identifiers` are only names, but params have types too?

Reply:
Yea, in fact some of our other instances of `Identifier`s actually should have an associated type as well. It's not really clear what it will eventually look like at this point until we start type checking.